src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/Overview/ViewAllOtherEvidenceOverview.cshtml

```diff
@@ -126,6 +127,9 @@
                         </tbody>
                     </table>
                     <br />
+                    @{
+                        Html.RenderPartial("~/Views/Shared/_EvidenceNotesPagerPartial.cshtml", Model.EvidenceNotesDataList, new ViewDataDictionary() { { "pagerRouteValues", new { tab = Model.ActiveOverviewDisplayOption.ToDisplayString(), selectedComplianceYear = Model.ManageEvidenceNoteViewModel.SelectedComplianceYear } } });
```

Comment:
@NateHogan  How does this work with the filters. I think if the user has filtered the results and they change page the selected filters should be retained